### ISSUE TITLE
Allays have a single tab. (0.9.beta)

### ISSUE THREAD

**Issue by Agent00Ming** [Andrew6rant/InventoryTabs#41]
One or multiple allays will share a single tab that cannot be interacted with. (No visible inventory slots)
![image](https://user-images.githubusercontent.com/107314235/181107744-b07515fd-d1f0-412b-8f68-b17e109ab487.png)

**Comment by Andrew6rant** [comment]
Yeah, currently duplicate entities show up as one tab. That is part of the reason why 0.9 is currently a beta. Block tabs use their coordinates to handle duplicates, but I can't do that for entities since they move. I'll fix this sometime in the future
